ApptentiveFeedback/ApptentiveFeedback/Additions/ApptentiveCallbacks.cs

```diff
@@ -0,0 +1,358 @@
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A String value.</param>
+        public static void AddCustomPersonData(String key, String value)
+        {
+            ApptentiveKit.Android.Apptentive.AddCustomPersonData(key, value);
+        }
+
+        /// <summary>
+        /// Add a custom data int to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A int value.</param>
+        public static void AddCustomPersonData(String key, int value)
+        {
+            Java.Lang.Integer intValue = Java.Lang.Integer.ValueOf(value);
+            Java.Lang.Number numberValue = (Java.Lang.Number)intValue;
+
+            ApptentiveKit.Android.Apptentive.AddCustomPersonData(key, numberValue);
+        }
+
+        /// <summary>
+        /// Add a custom data bool to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="value"></param>
+        public static void AddCustomPersonData(String key, bool value)
+        {
+            Java.Lang.Boolean javaValue = Java.Lang.Boolean.ValueOf(value);
+            ApptentiveKit.Android.Apptentive.AddCustomPersonData(key, javaValue);
+        }
+
+        /// <summary>
+        /// Remove a piece of custom data from the Person. 
+        /// </summary>
+        /// <param name="key">The key to remove.</param>
+        public static void RemoveCustomPersonData(String key)
+        {
+            ApptentiveKit.Android.Apptentive.RemoveCustomPersonData(key);
+        }
+
+        /// <summary>
+        /// Add a custom data String to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A String value.</param>
+        public static void AddCustomDeviceData(String key, String value)
+        {
+            ApptentiveKit.Android.Apptentive.AddCustomDeviceData(key, value);
+        }
+
+        /// <summary>
+        /// Add a custom data int to the Person. Custom data will be sent to the server, is displayed
+        /// in the Conversation view, and can be used in Interaction targeting.
+        /// </summary>
+        /// <param name="key">The key to store the data under.</param>
+        /// <param name="value">A int value.</param>
+        public static void AddCustomDeviceData(String key, int value)
+        {
```

Comment:
Same here. We should provide support to pass all the Number formats. If Xamarin doesn't support Number, you could add overloaded functions with Long, Float and Double.